shiftfsMgr/shiftfsMgr.go

```diff
@@ -38,92 +42,179 @@ import (
 var testingMode bool = false
 
 type mgr struct {
-	mpMap map[string][]string // maps each shiftfs markpoint to all it's associated container(s)
-	mu    sync.Mutex          // protects the mark point map
+	workDir string
+	cntrMap map[string][]string // container map (maps shiftfs mount request paths to the associated container(s))
```

Comment:
A couple of suggestions below ...

What about renaming 'cntrMap' to better reflect how this map is being indexed? The way it is now makes it a bit difficult to follow the code coz container-map can mean many different things, specially since we have 'mpMap' too which also misses the 'key' attribute in its name.

cntrMap -> mreqCntrMap
mpMap  -> mpMreqMap

Finally, the description field should talk about "container-ids" to differentiate it from the existing "containerInfo" or "containerState" structs.

Reply:
Good suggestion, done.